cli/src/test/java/hudson/cli/PlainCLIProtocolTest.java

```diff
@@ -126,6 +133,9 @@ void newop() throws IOException {
                 client.wait();
             }
         }
+        while (server.stdin.size() == 0) {
+            Thread.sleep(100);
+        }
```

Comment:
could it maybe use `await` from awaitility? (Already used in the `test` module for example)

Reply:
Yeah I guess. Not really any easier for a reader to understand IMO.